moda-contracts/src/interfaces/ICatalog.sol

```diff
@@ -0,0 +1,336 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.21;
+
+/**
+ * @title IRegistry
+ * @dev Interface for Registry
+ */
+interface ICatalog {
+    /**
+     * @dev Status of a registered track
+     */
+    enum TrackStatus {
+        PENDING,
+        VALIDATED,
+        INVALIDATED
+    }
+
+    /**
+     * @dev Represents a registered track
+     * TrackStatus - The status of the track (pending, validated, invalidated)
```

Comment:
NIT: `The registration state of the track`.  The other sounds redundant

Reply:
Fixed